packages/@react-stately/tooltip/src/useTooltipTriggerState.ts

```diff
@@ -119,6 +120,10 @@ export function useTooltipTriggerState(props: TooltipTriggerProps = {}): Tooltip
     }
   };
 
+  useEffect(() => {
```

Comment:
we can use `useEffectEvent` instead of managing this ref ourselves

Reply:
Are you sure, that's a good idea?
From the doc
```
Effect Events are very limited in how you can use them:

**Only call them from inside Effects.**
Never pass them to other components or Hooks.
```
we call 'close' not inside effect

Reply:
we have one from before those limitations were involved and because we have to support it all the way back to react 16, https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/utils/src/useEffectEvent.ts which I think is doing the same thing as you've done here

but no worries, can replace later